### PR TITLE
add test.cpp

### PR DESCRIPTION
**Pull request by hetong07**
request review

### CODE REVIEW

File: src/test.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+
+
+int main(int argc, char const *argv[])
```
Comment (zainryan) [inline src/test.cpp]: Do not write main function. Make it testable.

Comment (zainryan) [reply src/test.cpp]: Also please link your frequently-used email to your github account so that you'd receive timely notifications.

Comment (hetong07) [reply src/test.cpp]: solved the link email problem

File: src/tonghe_cal.cpp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
```
Comment (zainryan) [inline src/tonghe_cal.cpp]: You don't need this header.

File: src/tonghe_cal.h [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#ifndef TONGHE_H_
```
Comment (zainryan) [inline src/tonghe_cal.h]: You could use #pragma once instead which is much more simpler.

**Pull request merged**